exchanges/bybit/futures_type.go

```diff
@@ -51,32 +55,32 @@ type FuturesCandleStickWithStringParam struct {
 
 // SymbolPriceTicker stores ticker price stats
 type SymbolPriceTicker struct {
-	Symbol                 string              `json:"symbol"`
-	BidPrice               float64             `json:"bid_price,string"`
-	AskPrice               float64             `json:"ask_price,string"`
-	LastPrice              float64             `json:"last_price,string"`
-	LastTickDirection      string              `json:"last_tick_direction"`
-	Price24hAgo            float64             `json:"prev_price_24h,string"`
-	PricePcntChange24h     float64             `json:"price_24h_pcnt,string"`
-	HighPrice24h           float64             `json:"high_price_24h,string"`
-	LowPrice24h            float64             `json:"low_price_24h,string"`
-	Price1hAgo             float64             `json:"prev_price_1h,string"`
-	PricePcntChange1h      bybitNumericalValue `json:"price_1h_pcnt"`
-	MarkPrice              float64             `json:"mark_price,string"`
-	IndexPrice             float64             `json:"index_price,string"`
-	OpenInterest           float64             `json:"open_interest"`
-	OpenValue              bybitNumericalValue `json:"open_value"`
-	TotalTurnover          bybitNumericalValue `json:"total_turnover"`
-	Turnover24h            float64             `json:"turnover_24h,string"`
-	TotalVolume            float64             `json:"total_volume"`
-	Volume24h              float64             `json:"volume_24h"`
-	FundingRate            float64             `json:"funding_rate,string"`
-	PredictedFundingRate   bybitNumericalValue `json:"predicted_funding_rate"`
-	NextFundingTime        string              `json:"next_funding_time"`
-	CountdownHour          int64               `json:"countdown_hour"`
-	DeliveryFeeRate        bybitNumericalValue `json:"delivery_fee_rate"`
-	PredictedDeliveryPrice bybitNumericalValue `json:"predicted_delivery_price"`
-	DeliveryTime           string              `json:"delivery_time"`
+	Symbol                 string                  `json:"symbol"`
+	BidPrice               float64                 `json:"bid_price,string"`
+	AskPrice               float64                 `json:"ask_price,string"`
+	LastPrice              float64                 `json:"last_price,string"`
+	LastTickDirection      string                  `json:"last_tick_direction"`
+	Price24hAgo            float64                 `json:"prev_price_24h,string"`
+	PricePcntChange24h     float64                 `json:"price_24h_pcnt,string"`
+	HighPrice24h           float64                 `json:"high_price_24h,string"`
+	LowPrice24h            float64                 `json:"low_price_24h,string"`
+	Price1hAgo             float64                 `json:"prev_price_1h,string"`
+	PricePcntChange1h      convert.NullableFloat64 `json:"price_1h_pcnt"`
+	MarkPrice              float64                 `json:"mark_price,string"`
+	IndexPrice             float64                 `json:"index_price,string"`
```

Comment:
Just curious on the selective application of the type, exposing the same issue on "" being returned. Any reason not to change all float64 ,strings to convert.NullableFloat here?

Reply:
No Idea, might be a cherry pick issue between updated structs or just made in error. Good spotting though!